models/generated/workloads/vm.go

```diff
@@ -0,0 +1,103 @@
+package workloads
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"math"
+	"net"
+
+	"github.com/rs/zerolog/log"
+	schema "github.com/threefoldtech/tfexplorer/schema"
+)
+
+var _ Workloader = (*VirtualMachine)(nil)
+var _ Capaciter = (*VirtualMachine)(nil)
+
+type VirtualMachine struct {
+	ReservationInfo `bson:",inline"`
+
+	Flist     string   `bson:"flist" json:"flist"`
```

Comment:
No, Our vms will not be used "with" flists instead they will have a specific names for example:
- "ubuntu"
- "archlinux"
- etc ...

In ZOS the name will map to one official flist that is maintained by us, we will not allow running user flists.

Reply:
Updated to fetch the VM flists from the [tf-official-repo](https://hub.grid.tf/tf-official-vms)